android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -80,6 +80,9 @@
   private SwipeRefreshWidget mSwipeRefreshWidget;
   private boolean mDebugViewEnabled = false;
   private boolean mSecondScanComplete;
+  private boolean mFirstTime = false;
```

Comment:
Shouldn't this be initialized to true?  Then you won't have to set it in onCreateView